scripts/_settings.sh

```diff
@@ -40,7 +41,7 @@ fi
 if [ ! -d "$SCRIPTDIR/vendor/bash_ini_parser" ]; then
   echo ""
   echo "Installing Bash ini parser..."
-  git clone https://github.com/rudimeier/bash_ini_parser.git "$SCRIPTDIR/vendor/bash_ini_parser"
+  git clone https://github.com/pmgouveia/bash_ini_parser.git "$SCRIPTDIR/vendor/bash_ini_parser"
```

Comment:
Great! Tests are failing at that PR tho.

If you'd like to test that change on Ubuntu/Debian, you can spin up a server for free at new DigitalOcean account with this referral: https://m.do.co/c/6dc078966c9c (10$ for you, 25$ for Trustroots)